specification/redis/resource-manager/Microsoft.Cache/stable/2022-06-01/redis.json

```diff
@@ -1947,6 +1947,10 @@
               "description": "The max clients config",
               "readOnly": true
             },
+            "notify-keyspace-events": {
+              "type": "string",
+              "description": "The KeySpace Events which should be monitored."
```

Comment:
Doc capitalization can be as ordinary english

Reply:
thanks for the review - to clarify, do you mean?

```suggestion
              "description": "The keyspace events which should be monitored."
```

Reply:
Yup, that works